### DISCUSSION TITLE
Help with custom kernel modifications

### DISCUSSION THREAD

**Discussion by SeeRich** [pythops/jetson-image#148] [General]
I honestly am not sure where to start with this question because I am not super experienced with the linux kernel or containerization (podman) but I will do my best to describe the situation.

I am working with the Jetson Nano and am interested in flashing Ubuntu 20.04. I can successfully run the code in this repo and flash jetson nano boards that work fine in the devkit carrier. However, I need to build for a custom carrier board that requires kernel + device tree patches to work.

The custom carrier board setup is not containerized at all and is a set of bash scripts that follow instructions similar to here: https://docs.nvidia.com/jetson/archives/r35.3.1/DeveloperGuide/text/SD/Kernel/KernelCustomization.html

My question is how can I encorporate custom kernel patches into the build workflow contained here?

**Comment by pythops** [comment]
Good question ! 
I never did it on nvidia jetson but it's something I can look at for sure

**Reply by SeeRich** [reply]
That would be amazing, thank you! 

For now, I am going to attempt to create a rootfs containerfile for my existing set of bash scripts (based on ubuntu 18.04) and reuse `Containerfile.image.l4t32`.

**Comment by pythops** [comment]
Even though it's interesting I think I will close this one cause it's not really related to any issue of this repo. Feel free to reach out via email or Discord if you wanna discuss it aside
